### ISSUE TITLE
 SecDebugLog does not work. 

### ISSUE THREAD

**Issue by olekstumedia** [SpiderLabs/ModSecurity-nginx#17]
Has this in my site vhost config file for Nginx. By some reason debug log does not work ... although audit one works. What could be the reason? how to troubleshoot such issues(( It looks like a bug

###################################
```
  modsecurity on;

  location / {
_##  root /var/www/html;
##  modsecurity_rules_file /etc/nginx/modsecurity-3/nginx-modsecurity-complete.conf;
##   modsecurity_rules_file /etc/nginx/modsec3/modsecurity.conf;_
  modsecurity_rules '
    SecRuleEngine On
    SecDebugLog /tmp/modsec_debug.log
    SecDebugLogLevel 9
    SecRule ARGS "@contains test" "id:1,phase:2,t:trim,block"
  ';
  }
```
#####################################

**Comment by olekstumedia** [comment]
and it looks like it does not work... I think the rule should block such URLs like https://mysite.com/test

**Comment by p0pr0ck5** [comment]
I see the same issue. The nginx worker process is actually writing debug log lines to stdout. Additionally, the test executables provided with libmodsecurity also write to stdout even when SecDebugLog is configured, though I see several `open()` calls to the configured debug log file path. Perhaps a bug in `debug_log_writer` or `debug_log_writer_agent`?

**Comment by zimmerle** [comment]
Hi @p0pr0ck5,

I am looking into this bug at this very moment. I will have more information in a few minutes.

**Comment by zimmerle** [comment]
Hi @p0pr0ck5 and @olekstumedia,

Took some time but the problem should be fixed by: SpiderLabs/ModSecurity@678a97d0f749d7a8474543e8142d18bd902fe266

**Comment by vishaalgc** [comment]
Im facing similar issue, Not able to find anything from SecDebugLog in the defined location. 0 lines in the file. Is this issue fixed ? if so whats the solution ? @olekstumedia @zimmerle
